hatch/src/cli/commands/update.rs

```diff
@@ -1,61 +1,35 @@
+use HatchResult;
 use clap::{ App, SubCommand, Arg, ArgMatches };
 use cli::commands::Command;
-
+use cli::commands::ops::ProjectOps;
 use yaml;
+use project::Project;
+use hatch_error::HatchError;
 
-use yaml_rust::Yaml;
+struct AmbiguousUpdater;
+struct ExplicitUpdater;
 
-use project::{ Project, LibraryKind, ProjectKind };
+impl ProjectOps for AmbiguousUpdater {
```

Comment:
Same comment about the meaning of Ambiguous as with the Builder.

Reply:
After our discussion we are going to use the word Inferred instead of Ambiguous.